helix-term/Cargo.toml

```diff
@@ -65,6 +65,8 @@ serde = { version = "1.0", features = ["derive"] }
 grep-regex = "0.1.9"
 grep-searcher = "0.1.8"
 
+is-terminal = "0.3.0"
```

Comment:
Please use the method available in crossterm rather than importing another dependency https://docs.rs/crossterm/latest/crossterm/tty/trait.IsTty.html

Reply:
I didn't realize crossterm had implemented this. Thanks !